shared_utils.py

```diff
@@ -117,7 +118,15 @@ def check_styles(soup: bs4.BeautifulSoup) -> None:
         warn('style_no_intro')
     if not get_elem_containing_text(soup, 'h1', 'references'):
         warn('style_no_refs')
-
+    # Check every numbered reference appears in the text in square brackets
+    dom = etree.HTML(str(soup))
+    ref_li = dom.xpath("//h1[contains(text(), 'REFERENCES')]/following-sibling::ol/li | //h1[.//*[contains(text(), 'REFERENCES')]]/following-sibling::ol/li") 
```

Comment:
This doesn't work for all DOCX files, in a way that I think will confuse authors. The _Heading 1_ style in the template forces uppercase character display, but the text of "References" may be secretly lowercase. When that happens, the `contains` doesn't match and it shows all references as mismatching.

Reply:
So case insensitive match on "References" would be OK?

Reply:
Done - in XPath 1, the way to do it is normalize to either upper or lowercase, then check for that.